addons/account_peppol/models/account_move_send.py

```diff
@@ -240,4 +238,4 @@ def action_what_is_peppol_activate(self, moves):
             }
             return action
         else:
-            return moves.action_send_and_print()
+            return {}
```

Comment:
To modify to reopen the existing wizard if possible, will avoid the double HTTP call.
Otherwise good to go, good work :+1: